src/OrchardCore/OrchardCore.Abstractions/Extensions/Features/ITypeFeatureProvider.cs

```diff
@@ -10,8 +10,39 @@ namespace OrchardCore.Environment.Extensions
     /// </summary>
     public interface ITypeFeatureProvider
     {
+        /// <summary>
+        /// Gets the extension for the specified dependent type.
+        /// </summary>
+        /// <param name="dependency"></param>
+        /// <returns></returns>
```

Comment:
Remove empty `<returns></returns>`s and empty parameter docs.

Reply:
Done.